app/assets/stylesheets/shared/header.scss

```diff
@@ -105,16 +106,13 @@
   width: 100%;
   border-bottom: 1px solid $gray-200;
 
-  @media (min-width: 576px) {
+  @media (min-width: 992px) {
+    min-width: 350px;
```

Comment:
To avoid weird spacing, we need the podcast switcher to be full width until it hits 992px (or the large screensize). it may seem like a lot, but this works best!